### PR TITLE
Azure Network Swagger June changes merged to .NET SDK 19.1.0

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by MikhailTryakhov** [comment]
+@dsgouda to review SDK PR

**Comment by MikhailTryakhov** [comment]
+@lmazuel can you please take a look?

File: src/SDKs/Network/Management.Network/Generated/IVirtualNetworkGatewayConnectionsOperations.cs [hunk 1]
```diff
@@ -129,7 +129,7 @@ public partial interface IVirtualNetworkGatewayConnectionsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<VirtualNetworkGatewayConnectionListEntity>> UpdateTagsWithHttpMessagesAsync(string resourceGroupName, string virtualNetworkGatewayConnectionName, TagsObject parameters, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<VirtualNetworkGatewayConnection>> UpdateTagsWithHttpMessagesAsync(string resourceGroupName, string virtualNetworkGatewayConnectionName, TagsObject parameters, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Generated/IVirtualNetworkGatewayConnectionsOperations.cs]: This is a breaking change in the sdk. Please bump the package version to `19.1.0`

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("19.0.2.0")]
-[assembly: AssemblyFileVersion("19.0.2.0")]
+[assembly: AssemblyVersion("19.0.3.0")]
+[assembly: AssemblyFileVersion("19.0.3.0")]
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: `AssemblyFileVersion` must be updated, `AssemblyVersion` remains the same.

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 3]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.0.2-preview</Version>
+		<Version>19.0.3-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by MikhailTryakhov** [comment]
@dsgouda  please take a look for changes please

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("19.0.2.0")]
-[assembly: AssemblyFileVersion("19.0.2.0")]
+[assembly: AssemblyVersion("19.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: `AssemblyVersion` remains the same for minor version updates, only the `AssemblyFileVersion` changes

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 5]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.0.2-preview</Version>
+		<Version>19.1.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
A couple of comments apart looks good

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**